release_notes/45-release-notes.adoc

```diff
@@ -376,13 +377,31 @@ The following section provides information about removed features listed in the
 * Direct upgrades from {product-title-short} version 3.74 or earlier to version 4.5 are no longer supported. To upgrade to version 4.5 or later, you must first upgrade to version 4.4.
 
 [id="bug-fixes_{context}"]
-== Bug fixes
-
-[id="resolved-in-version-450_{context}"]
-=== Resolved in version 4.5.0
+== Bug fixes in version 4.5.0
 
 *Release date*: 24 July 2024
 
 * Previously, a bug prevented users without administrator privileges from accessing the listening endpoint data for clusters and namespaces for which they had `read` permission. With this update, users with `read` permission can now receive data from the listening endpoints service.
 
+[id="about-release-451_{context}"]
+== About release 4.5.1
+
+This release includes notable technical changes and bug fixes.
+
+[id="changes-in-version-451_{context}"]
+=== Changes in version 4.5.1
+
+* The network graph and the network policy generator in the {product-title-short} portal were updated to clarify that `AdminNetworkPolicy` and `BaselineAdminNetworkPolicy` resources are not taken into consideration in the network graph or during network policy generation. For example, flows to or from IP addresses not belonging to private IP address space, as defined in RFC 1918, are shown as external on the graph.
+
+* The field `error` returned for failed API calls has been deprecated, and will be removed in a future release. Instead of using the `error` field, use the `message` field. The `message` field contains the same information as the `error` field.
+
+[id="resolved-in-version-451_{context}"]
+=== Bug fixes in version 4.5.1
+
+*Release date:* 8 August 2024
+
+* When upgrading to version 4.5 with the {product-title-short} Operator, the upgrade failed with a `central-db` error for the `uni_compliance_integrations_clusterid` constraint. This problem has been fixed.
+* After upgrading to version 4.5.0, for nodes with more than 64 cores, Collector displayed an error for the per-CPU ring buffer dimension. This problem has been fixed.
```

Comment:
It would be more correct to say just "ring buffer size". Technically speaking a ring buffer doesn't have to be per-CPU:
```suggestion
* After upgrading to version 4.5.0, for nodes with more than 64 cores, Collector displayed an error about the not allowed ring buffer size. This problem has been fixed.
```